src/main/java/net/minecraftforge/client/model/ModelLoader.java

```diff
@@ -24,9 +24,7 @@
 import java.awt.image.BufferedImage;
 import java.util.ArrayList;
 import java.util.Collection;
-import java.util.Collections;
```

Comment:
All of these import cleanups aren't really necessary for this PR, could probably amend it and revert the changes to them